pkg/eks/eks_test.go

```diff
@@ -408,6 +408,16 @@ var _ = Describe("EKS API wrapper", func() {
 			expectError:       false,
 			supports:          false,
 		}),
+		Entry("with 1.15", &managedNodesSupportCase{
+			kubernetesVersion: "1.15",
+			expectError:       false,
+			supports:          true,
+		}),
+		Entry("with 1.16", &managedNodesSupportCase{
```

Comment:
so 1.16 is coming soon as well ? :100:

Reply:
Not that I know of, this is just to ensure the validation function works for all newer versions.